src/Suggestor/Suggestor.ts

```diff
@@ -37,8 +45,15 @@ export function makeDefaultSuggestionBuilder(
             addRecurrenceSuggestions(line, cursorPos, settings, symbols.recurrenceSymbol, dataviewMode),
         );
 
-        // Step 3: add task property suggestions ('due', 'recurrence' etc)
-        suggestions = suggestions.concat(addTaskPropertySuggestions(line, cursorPos, settings, symbols, dataviewMode));
+        // Step 3: add dependecy suggestions
+        suggestions = suggestions.concat(
+            addBlockedBySuggestions(line, cursorPos, settings, symbols.dependsOnSymbol, allTasks, taskToSuggestFor),
+        );
+
+        // Step 4: add task property suggestions ('due', 'recurrence' etc)
+        suggestions = suggestions.concat(
+            addTaskPropertySuggestions(line, cursorPos, allTasks, settings, symbols, dataviewMode),
+        );
```

Comment:
Could you say the thinking behind putting dependencies above dates?

I'm genuinely interested in understanding.

I'm wondering, as I think that many more people will add dates than dependencies - and the number of down-arrows needed to reach particular fields will likely be in people's muscle memory.

Reply:
I actually just added it, after my Personal Opinion/Preference.
Personally, I don't Work with Dates, if I don't have to.
Reason being if I Plan Everything out, and then just one Milestone/Task, gets delayed. (Which Unfortunate happens more often than i want to Admit =)
I would need to move Every Single Date, to the new Delayed Date. (Lots of Work).
For me Personally it Makes more sense, to Define the Relation via Dependencies, then Add Duration to the Task, and Ideally the Date gets Calculated and Updated from that.
(I do most of it for Personal Fun, and don't Have a Deadline to Keep)

But this is Use Case, Projekt Planning.
For Quick Temporary Tasks Planning, i do add Dates.

So again it is Personal Preference.

Will Happily move it to another Place.
Where do you suggest it should go?

Reply:
It turns out that the new order is necessary to ensure that the 'Depends on' option puts suggested tasks before other types of suggestion.

Resolving.